### ISSUE TITLE
Incorrect park rating

### ISSUE THREAD

**Issue by sudofox** [Basssiiie/OpenRCT2-ParkRatingInspector#20]
[WrongRating.zip](https://github.com/Basssiiie/OpenRCT2-ParkRatingInspector/files/9551323/WrongRating.zip)

Park rating shown in UI: 0

Park rating shown in inspector: ~900-something

**Comment by Basssiiie** [comment]
Hey @sudofox , thank you for your bug report! I have reproduced the issue and discovered this is a bug in OpenRCT2 itself. The bug occurs when the park has more than 32.767 guests (the maximum for a 16 bit value). The bug could also occurs with a total ride excitement or intensity in the whole park.

I have submitted a fix for this to the OpenRCT2 source code, so it should eventually make it into the game. There will be no need to update the plugin (as that one still reported the correct rating) The fix in the game itself can be tracked here: OpenRCT2/OpenRCT2#18026

If you have any further questions, let me know! :)
